### DISCUSSION TITLE
Community for discussion?

### DISCUSSION THREAD

**Discussion by zqevans** [magenta/ddsp#281] [Ideas]
In the last month or so, I've gotten really into machine learning and deep learning for music and audio, largely inspired by DDSP. I have a lot of ideas of things I think I could use this library for, but I don't really have the ML experience to put these into practice. I'm doing my best to learn as quickly as possible, but it would really help me if I had other people to talk to who are also working in this space.

Is there an online community somewhere where people are talking about this kind of thing? Something more like Discord/Slack than Reddit (more real-time conversation). I'd love to have somewhere I can post questions and ideas and have conversations about feasibility and implementation details. Right now this "issues" section on this repo is where I'm seeing the most conversation about DDSP, which really doesn't seem like the right forum for general discussion unrelated to issues with the repo.

**Comment by jesseengel** [comment]
Hi, glad you're enjoying the repo! I agree that the issues section isn't the best format for discussions. Inspired by this, I'm going to try an experiment. 

It's not quite Discord/Slack (sorry, can only maintain so many things), but I've enabled the new "Discussions" feature of Github for this repo. You should be able to see it in the tabs above. Let's try it out as a place to share all things DDSP, connect with each other, share what we've been working on, and discuss. I know these things can get out of hand, so let's give it a trial run and see how it goes.

**Reply by zqevans** [reply]
Thanks! I'm really excited about this and I'm hoping to be able to learn from others.
